app/views/assignments/_read.html.erb

```diff
@@ -104,15 +104,4 @@
       <% end -%>
     </div>
   <% end %>
-
-  <h3><%= t('assignment.required_files_read') %></h3>
```

Comment:
You are right, it is used by one more view. I still need to make changes to this file because students will not submit files for peer-reviews, but I will make sure both views have the desired behaviour.